packages/bolt-storage-file/README.md

```diff
@@ -0,0 +1,25 @@
+# Bolt File Storage
+
+Bolt [conversation store](https://slack.dev/bolt/concepts#conversation-store) that uses your file system as a database. Useful for retaining conversation state through restarts of your App in development.
+
+## Install
+
+```bash
+# Yarn
+yarn add --dev @slack-wrench/bolt-storage-file
+
+# npm
+npm install --save-dev @slack-wrench/bolt-storage-file
+```
+
+## Usage
+
+```typescript
+import { App } from '@slack/bolt';
+import FileStore from '@slack-wrench/bolt-storage-file';
+
+const databasePath = '.db'; // defaults to ".db"
+const fileStore = new FileStore(databasePath);
```

Comment:
wouldn't the happiest path jusrt be 
```js
const fileStore = new FileStore(); 
```

? 

my guess is most people wouldn't customize. don't feel strongly here, just wondering how easy we want the initial example to be.

Could have a different header showing how to customize the usage by passing in a `databasePath`

Reply:
Having a whole section for `path` seems unnecessary to me. I'll try something else to communicate it.